pkg/k8s/metricsclient.go

```diff
@@ -0,0 +1,21 @@
+package k8s
+
+import (
+	"fmt"
+
+	"k8s.io/metrics/pkg/client/clientset/versioned/typed/metrics/v1beta1"
+	ctrl "sigs.k8s.io/controller-runtime"
+)
+
+var metricsClientLog = ctrl.Log.WithName("metricsclient")
+
+// InitMetricsClient initializes metrics client
+func InitMetricsClient(mgr ctrl.Manager) (v1beta1.PodMetricsesGetter, error) {
```

Comment:
Can we use `PodMetricsClient` or something similar to name this file/funcs... we have way to many different metrics in KEDA, so we should be more explicit. Also extending the docs/comment would be great